Packs/MajorBreachesInvestigationandResponse/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Rapid Breach Response",
     "description": "This content Pack helps you collect, investigate, and remediate incidents related to major breaches.",
     "support": "xsoar",
```

Comment:
Manually add the pack as an optional dependency

Reply:
Done